.kitchen.yml

```diff
@@ -1,8 +1,6 @@
 ---
 driver:
   name: vagrant
-  network:
-    - ["forwarded_port", {guest: 8983, host: 8888}]
```

Comment:
i find it useful during `vagrant converge`, any specific reason for not keeping it?